### PR TITLE
Added tooltips to feedback buttons

### PR DESCRIPTION
**Pull request by dwnoble**
Screenshot
tooltips

### CODE REVIEW

File: static/js/components/nl_feedback.tsx [hunk 1]
```diff
@@ -33,117 +34,86 @@ interface NlChartFeedbackPropType {
...
+  },
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.DEMOTE,
+    content: <>&#11015;</>,
+    tooltip: "The chart should be demoted",
+  },
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.FACE_PALM,
+    content: <>&#129318;&#127995;</>,
+    tooltip: "The chart is embarassing or inappropriate for the the query",
```
Comment (pradh) [inline static/js/components/nl_feedback.tsx:61]: nit: typo

**Review by pradh — APPROVED** [review]
Nice cleanup!

File: static/js/components/nl_feedback.tsx (hunk 1 above)
Comment (dwnoble) [reply static/js/components/nl_feedback.tsx]: good catch- fixed!

Comment (pradh) [reply static/js/components/nl_feedback.tsx]: Hmm the checked in code has `the the` - https://github.com/datacommonsorg/website/blob/master/static/js/components/nl_feedback.tsx#L61 ?

**Pull request merged**